aries-site/src/examples/templates/list-views/ListNameOptionActionExample.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Box, List, Menu, Text } from 'grommet';
+import { More } from 'grommet-icons';
+
+const data = [
+  {
+    name: 'Server 1',
+    option: 'Active',
+  },
+  {
+    name: 'Server 2',
+    option: 'Inactive',
+  },
+  {
+    name: 'Server 3',
+    option: 'Active',
+  },
+  {
+    name: 'Server 4',
+    option: 'Active',
+  },
+  {
+    name: 'Server 5',
+    option: 'Inactive',
+  },
+];
+
+export const ListNameOptionActionExample = () => {
+  return (
+    <List
+      data={data}
+      action={item => (
+        <Box direction="row" align="center" gap="medium">
+          <Text>{item.option}</Text>
```

Comment:
These don't show up in Figma, but wondering about the associating color indicator along side status:
![Screen Shot 2020-07-06 at 2 24 20 PM](https://user-images.githubusercontent.com/1756948/86638433-84a9b180-bf94-11ea-88ac-2ba2b595d618.png)

Reply:
I like this idea. I will adjust one example to show this.